assets/js/atomic/blocks/product-elements/image/style.scss

```
@@ -55,5 +53,10 @@
 }
 
 .is-loading .wc-block-components-product-image {
-	@include placeholder();
+	@include placeholder($include-border-radius: true);
```

Should this be `false`?
```suggestion
	@include placeholder($include-border-radius: false);
```